src/main/java/ch/njol/skript/expressions/ExprLevelProgress.java

```diff
@@ -36,7 +36,7 @@
  * @author Peter Güttinger
  */
 @Name("Level Progress")
-@Description({"The progress a player has made until the next level. Remember that this value is between 0 and 1, not 0 and 100!",
+@Description({"The progress a player has made until he reaches the next level. Remember that this value is between 0 and 1, not 0 and 100!",

```

Comment:
```suggestion
@Description({"The progress a player has made until they reach the next level. Remember that this value is between 0 and 1, not 0 and 100!",
```
Still though, I find the whole sentence confusing